hydroshare/local_settings.template

```diff
@@ -191,3 +192,16 @@ HSWS_API_TOKEN = "d1caabf3059ef7ffd99d890aba57100da2e0efc8"
 HSWS_TIMEOUT = 3
 HSWS_PUBLISH_URLS = False
 HSWS_ACTIVATED = False
+
+# Uncomment the following code if you want to suppress some warnings on a local laptop
+# TIME_ZONE = 'Etc/UTC'
+# USE_TZ = False
+
+"""
+Silence ForeignKey Warnings:
+hs_tracking.Visitor.user:(fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
+hydroshare | HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
+hydroshare | security.PasswordExpiry.user: (fields.W342) Setting unique=True on a ForeignKey has the same effect as
+using a OneToOneField.
+"""
+# SILENCED_SYSTEM_CHECKS = ["mezzanine.core.W03", "fields.W342"]
```

Comment:
These are useful for developers often in the logs/console. Note they are commented out by default so the behavior of creating a local_settings.py from the template doesn't add a "change" to the former state the developer is familiar with.